src/components/NavBar/NavBar.jsx

```diff
@@ -0,0 +1,105 @@
+                    <li className="nav-main-li border-[1px] rounded border-color-grey px-2 hover:bg-color-highlight-orange hover:text-primary-bg hover:border-color-highlight-orange hover:shadow hover:shadow-orange-600">
+                        <Link to="/" className="nav-main-item nav-btn-login">
+                            <i className="fal fa-sign-out"></i>
+                        </Link>
+                    </li>
+
+                    {/* <li className="nav-main-li">
+                        {!auth ? (
+                            <Link to="/login" className="nav-main-item nav-btn-login">
+                                Login
+                            </Link>
+                        ) : (
+                            <div className="nav-main-item nav-btn-login" onClick={signOutHandler}>
+                                Logout
+                            </div>
+                        )}
+                    </li> */}
+                </ul>
+
+                {/* {Nav middle mobile} */}
+                {/* <ul className="nav-main-ul nav-main-mobile">
+                    <li className="nav-main-li">
+                        <label for="nav-main-search" className="nav-item-search">
+                            <input
+                                className="nav-item-search-input"
+                                type="text"
+                                placeholder="Looking for something?"
+                                name="nav-search"
+                            />
+                            <i className="far fa-search nav-main-middle-icn"></i>
+                        </label>
+                    </li>
+                </ul> */}
+
+                {/* Nav right Mobile*/}
+                {/* <ul className={`${navHamburger} nav-main-ul`}>
+                    <li className="nav-main-li-res">
+                        {!auth ? (
+                            <Link
+                                to="/login"
+                                className="nav-main-item nav-btn-login"
+                                onClick={toggleHamburger}
+                            >
+                                Login
+                            </Link>
+                        ) : (
+                            <div
+                                className="nav-main-item nav-btn-login"
+                                onClick={() => {
+                                    toggleHamburger();
+                                    signOutHandler();
+                                }}
+                            >
+                                Logout
+                            </div>
+                        )}
+                    </li>
+                    <li className="nav-main-li-res">
+                        <Link to="/" className="nav-right-btn" onClick={toggleHamburger}>
+                            Explore
+                        </Link>
+                    </li>
+                </ul> */}
```

Comment:
Commented code.

Reply:
Will remove this. Still have a bit of work left with the navbar which is why I kept the commented code.